packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1787,14 +1787,14 @@ test('mutable liquidity triggers and interest', async t => {
     ),
   );
 
-  aliceUpdate = await E(aliceNotifier).getUpdateSince();
+  aliceUpdate = await E(aliceNotifier).getUpdateSince(aliceUpdate.updateCount);
   t.deepEqual(aliceUpdate.value.debtSnapshot.debt, aliceRunDebtLevel);
 
   await manualTimer.tick();
   // price levels changed and interest was charged.
 
   // expect Alice to be liquidated because her collateral is too low.
-  aliceUpdate = await E(aliceNotifier).getUpdateSince();
+  aliceUpdate = await E(aliceNotifier).getUpdateSince(aliceUpdate.updateCount);
```

Comment:
I think adding `updateCount` to these calls is acceptable, though I'm not sure it's an improvement.  

Using `getUpdateSince` without an argument can be more robust in a test because the test won't wait in vain when something changes. It'll return the existing state, which should be tested.

Reply:
I spoke to this above but for review hygiene: it's better to be more explicit about whether a new state is expected, especially due to the interleaving of async calls that are common in our code.